pkg/stats/monitor.go

```diff
@@ -418,7 +424,7 @@ func (m *Monitor) GetAvailableMemory() float64 {
 	defer m.mu.Unlock()
 
 	if m.cpuCostConfig.MaxMemory == 0 {
-		return math.MaxFloat64
+		return float64(memory.FreeMemory()) / gb
```

Comment:
Does is FreeMemory cgroup aware? Does it return the total amount of memory on the system/assigned to the cgroup, or the amount of memory allocatable in the system?